### PR TITLE
CART-986: CaRT ghost_rank test fails on "crt_group_rank() Self rank was not set yet"

### PR DESCRIPTION
**Pull request by mallove79**
ghost_rank file extension was (accidentally?) changed from .py to .ypy.  Avocado
won't find it without the .py extension.

Quick-functional: true
Allow-unstable: true

Signed-off-by: Ethan Mallove <ethanx.a.mallove@intel.com>

### CODE REVIEW

**Comment by sylviachanoiyee** [comment]
@mallove79 , the test failed. Can you look into the failure? Either 
1. fix the test in this pr
2. file a ticket with the issue and skip the test in this pr

**Review by sylviachanoiyee — DISMISSED** [review]

**Comment by sylviachanoiyee** [comment]
@mallove79 , can you mention that the test is skipped and provide instrucitons on how to re-enable test in ticket
https://daosio.atlassian.net/browse/CART-986
?
thanks.

File: src/tests/ftest/cart/ghost_rank_rpc/one_node.py [hunk 1]
```diff
@@ -7,18 +7,19 @@
...
 
-class CartCoRpcOneNodeTest(CartTest):
+class CartGhostRankRpcOneNodeTest(CartTest):
     # pylint: disable=too-few-public-methods
     """Run CaRT ghost rank RPC test.
 
     :avocado: recursive
     """
 
+    @skipForTicket("CART-986")
```
[the remark this answers is not in the document]
Comment (phender) [reply src/tests/ftest/cart/ghost_rank_rpc/one_node.py:17]: You're missing an import:
```
from apricot import skipForTicket
```

**Review by phender — CHANGES_REQUESTED** [review]

**Comment by sylviachanoiyee** [comment]
NLT failed, probably need rebase. PR needs a ticket as well.

**Comment by mallove79** [comment]
> NLT failed, probably need rebase. 

I'll rebase after fixing the issue: _Failure is due to stale code in `test_rpc_to_ghost_rank` (circa the pmix days?) which doesn't properly use `crt_launch` and `crt_utils`, e.g., `crtu_srv_start_basic` should be called._

> PR needs a ticket as well.

[CART-986](https://daosio.atlassian.net/browse/CART-986) (title updated)

**Review by sylviachanoiyee — APPROVED** [review]

**Review by frostedcmos — APPROVED** [review]

**Pull request merged**